CONTRIBUTING.rst

```diff
@@ -1,56 +1,55 @@
-# James Webb Space Telescope Data Analysis Tool Notebooks
+============================
+Contribute to JDAT Notebooks
+============================
 
-   ``jdat_notebooks`` are one component of STScI's larger [Data Analysis Tools Ecosystem](https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis).
```

Comment:
I think this sentence should still be preserved somewhere. Maybe in the readme if not here?

Reply:
I have realized that this particular sentence already exists in the README. To avoid redundancy, I refrained from including it here.

Reply:
If it is in the readme, than that it ok.